### DISCUSSION TITLE
Inspecting on the Map object, the value will displayed as the Array, maybe the Object (key-value) UI will look more instinctively?

### DISCUSSION THREAD

**Discussion by Azurewarth0920** [vuejs/devtools#216] [Ideas]
Before:

<img width="230" alt="Screenshot 2024-02-02 at 18 29 29" src="https://github.com/vuejs/devtools-next/assets/22590005/69b8da63-6720-4b56-962e-ed6287f11242">

After:
<img width="142" alt="Screenshot 2024-02-02 at 18 34 54" src="https://github.com/vuejs/devtools-next/assets/22590005/fc0958e4-8653-4282-800d-ef54e3bdd4f3">

**Comment by webfansplz** [comment]
I think it makes sense, PR Welcome.
